CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ All notable changes to [camunda-dmn-js](https://github.com/camunda/camunda-dmn-j
 
 ___Note:__ Yet to be released changes appear here._
 
+* `DEPS`: update to `diagram-js-grid@1.1.0`
+* `DEPS`: update to `diagram-js@15.2.1`
+* `DEPS`: update to `dmn-js@17.0.3`
+* `DEPS`: update to `dmn-js-properties-panel@3.6.0`
+
+### Key Changes in Modeling
+
+* `FEAT`: DRD editor `Canvas` is now a focusable element ([bpmn-io/diagram-js#662](https://github.com/bpmn-io/diagram-js/pull/662))
+* `FEAT`: keyboard binds implicitly ([bpmn-io/diagram-js#662](https://github.com/bpmn-io/diagram-js/pull/662))
+
+### Breaking Changes
+
+* Keyboard is now implicit, and canvas is focusable, cf. [bpmn-io/diagram-js#662](https://github.com/bpmn-io/diagram-js/pull/662))
```

Comment:
```suggestion
* Keyboard is now bound implicitly, and canvas is focusable, cf. [bpmn-io/diagram-js#662](https://github.com/bpmn-io/diagram-js/pull/662)). Calls to `keyboard.bind` and `keyboard.bindTo` now result with a descriptive console error and have no effect.
```

Reply:
Let's update camunda-bpmn-js changelog in that case: https://github.com/camunda/camunda-bpmn-js/blob/main/CHANGELOG.md#breaking-changes

Reply:
To be clear: I'd like us to anchor at "do not use these APIs anymore", rather than "you can continue to use them. 

I think the old changelog entries + descriptive error on mis-use were OK already. To simplify this message it could read:

```
* Keyboard is now bound implicitly, and canvas is focusable, cf. [bpmn-io/diagram-js#662](https://github.com/bpmn-io/diagram-js/pull/662)). Attempts to bind the keyboard to an explicit HTML element have no effect.
```